digicampipe/calib/camera/dl1.py

```diff
@@ -72,7 +73,21 @@ def calibrate_to_dl1(event_stream, time_integration_options, picture_threshold=7
 
             dl1_camera.on_border = border
 
-            dl1_camera.cleaning_mask = cleaning.dilate(geom=geom, mask=dl1_camera.cleaning_mask)
+            """
+            # repaired piece of code from Cyril. The commented version above leads to border_flag = 1 in almost all events.
```

Comment:
I must admit I have absolutely no idea about this part of the code. I guess @calispac should comment.

I believe you, when you say the previous version in the master was simply broken and this works now. I would be great  if you could add a test, which shows this is working now (and if the test would be thrown against the old version, it would show that it does not work).

But if that is too much work for now ... having working code is always better than having broken code. Thanks for fixing it.

Reply:
its fine although we should move this piece of code to a function I think